dist/yoast-seo.js

```diff
@@ -2481,6 +2492,11 @@ YoastSEO.StringHelper.prototype.stripAllTags = function( textString ) {
 	return textString;
 };
 
+YoastSEO.StringHelper.prototype.cleanRegex = function( textString ) {
```

Comment:
I don't know which regex this cleans, but to me it seems like this function sanitizes a string. @atimmer please rename accordingly.